x-pack/plugin/ml/src/main/java/org/elasticsearch/xpack/ml/action/TransportGetCalendarsAction.java

```diff
@@ -34,35 +31,18 @@ public TransportGetCalendarsAction(TransportService transportService, ActionFilt
 
     @Override
     protected void doExecute(Task task, GetCalendarsAction.Request request, ActionListener<GetCalendarsAction.Response> listener) {
-        final String calendarId = request.getCalendarId();
-        if (request.getCalendarId() != null && Strings.isAllOrWildcard(request.getCalendarId()) == false) {
-            getCalendar(calendarId, listener);
-        } else {
-            PageParams pageParams = request.getPageParams();
-            if (pageParams == null) {
-                pageParams = PageParams.defaultParams();
-            }
-            getCalendars(pageParams, listener);
+        final String[] calendarIds = Strings.splitStringByCommaToArray(request.getCalendarId());
```

Comment:
The GET request docs will need updating to add the standard description of a wildcard-able ID parameter

"comma-separated list of calendar Ids and wildcard expressions. You can get all by using `_all`, or `*` or by omitting the ID"